### ISSUE TITLE
Missing a step in the quick setup?

### ISSUE THREAD

**Issue by agamemnus** [react-boilerplate/react-boilerplate#1783]
````
    Clone this repo using git clone --depth=1 https://github.com/react-boilerplate/react-boilerplate.git
    Run npm run setup to install dependencies and clean the git repo.
    We auto-detect yarn for installing packages by default, if you wish to force npm usage do: USE_YARN=false npm run setup
    At this point you can run npm start to see the example app at http://localhost:3000.
    Run npm run clean to delete the example app.
````

Shouldn't this say somewhere that we need to change directories to `react-boilerplate`?

**Comment by KarandikarMihir** [comment]
That's true! Could you send a PR? Thanks for pointing that out.

**Comment by agamemnus** [comment]
Ok.

https://github.com/react-boilerplate/react-boilerplate/pull/1785

**Comment by rdworth** [comment]
Can this issue be closed since #1785 was merged to master?

(tip: in the future the PR body can include a magic verb and a # reference to the issue number. This will both create a link to the issue when the PR is created as well as auto close the issue when the PR is merged to master. See https://help.github.com/articles/closing-issues-via-commit-messages/ and https://github.com/blog/1506-closing-issues-via-pull-requests for more info)

**Comment by gihrig** [comment]
@rdworth Merging to master was an [error](https://github.com/react-boilerplate/react-boilerplate/blob/dev/CONTRIBUTING.md#pull-requests), and unfortunately, closing related issues with a "magic verb" does not work when merging to dev.

Although trivial doc changes (such as this) seem to be a valid exception, the [community practice ](https://github.com/react-boilerplate/react-boilerplate/blob/master/.github/PULL_REQUEST_TEMPLATE.md)has been that all changes go through dev branch.

Thanks for the input 😄 , Closing...
